src/app/shared/components/status-info-card/status-info-card.component.scss

```diff
@@ -26,16 +26,23 @@
 
 .application-info-description,
 .application-info-title {
-  flex: 1;
   max-width: max-content;
   line-height: 17px;
   color: black;
+  place-content: flex-start space-between;
```

Comment:
Why didn't you use tailwind classes in this case?

Reply:
because there are the same properties for both elements and there is no reason to dublicate it

Reply:
Yes, but you can just use the ready-made tailwind classes, so it shouldn't be a problem.

Reply:
i made that change in previous commit as you said